scripts/travis/build-all-in-one-image.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 BRANCH=${BRANCH:?'missing BRANCH env var'}
+GOARCH=$(go env GOARCH)
```

Comment:
Is this the right value? The shell script will read the host's arch, but the docker image should be independent of that.

Reply:
I've modified the defination a little bit to set default [GOARCH](url) variable to the host GOARCH, the target architecture can be overrided by passing architecture value to the script by:
`GOARCH=<target arch> ./scripts/travis/build-all-in-one-image.sh`.